environment.yml

```diff
@@ -11,7 +11,8 @@ dependencies:
   - cdsapi
   - cf-units
   - cf_xarray
-  - cmip6_preprocessing
+  - pip:
+    - git+https://github.com/jbusecke/cmip6_preprocessing.git
```

Comment:
You'll need to add that as a dockerfile run command b/c conda-lock does not support pip pkgs.

Reply:
Ah thanks. I added it in the bottom of the dockerfile, could you check if that is total nonsense?